mod_reforged/hooks/skills/perks/perk_nine_lives.nut

```diff
@@ -0,0 +1,40 @@
+::mods_hookExactClass("skills/perks/perk_nine_lives", function(o) {
+    local create = o.create;
+	o.create = function()
+	{
+        create();
+
+        // New Variables to control and tweak the behavior of this perk
+        this.m.MinHP <- 11;
```

Comment:
Please add these `MinHP` and `MaxHP` fields directly in the hook, before the `create` function. So, at the top of the hook:
```squirrel
::mods_hookExactClass("skills/perks/perk_nine_lives", function(o) {
    o.m.MinHP <- 11;
    o.m.MaxHP <- 15;
```